java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/model/Menu.java

```diff
@@ -19,29 +20,28 @@
   @JsonProperty("id")
   private String id;
 
-  @JsonProperty("Name")
+  @JsonProperty("restaurantId")
+  private UUID restaurantId;
+
+  @JsonProperty("name")
   private String name;
 
-  @JsonProperty("Description")
+  @JsonProperty("description")
   private String description;
-
-  @JsonProperty("TenantId")
-  private String restaurantId;
-
   @JsonProperty("Categories")
```

Comment:
If everything is changing to camel case then why not `Categories` too?

Reply:
fixed.